packages/manager/modules/support/src/tickets/new-ticket/issues-form/issues-form.controller.js

```diff
@@ -60,49 +79,65 @@ export default class SupportNewIssuesFormController {
     return filter(get(this.issue, 'selfCareResources'), { type: 'tip' });
   }
 
+  findServiceByName(name) {
+    return this.serviceTypes
+      .find(type => type.name.toLowerCase() === name);
```

Comment:
`type.name` is always defined ?

Reply:
Yes, and if it's not , it crashes, both are wanted behaviors